python/google/protobuf/internal/text_encoding_test.py

```diff
@@ -32,7 +32,10 @@
 
 """Tests for google.protobuf.text_encoding."""
 
-import unittest
+try:
+  import unittest2 as unittest
```

Comment:
Python 2.6 support - a bunch of the unit tests use new methods only added in Python 2.7. Rather than try to reimplement them, I just added the `unittest2` backport as a dependency for Python 2.6. See https://github.com/google/protobuf/issues/646#issuecomment-131269740 for the discussion around this.